Automattic/jetpack
# Boost Score Graph: Improve new graph

## Proposed changes:
* Instead of showing empty space before the first datapoint, show graph lines coming to the first data point.
* Fix: don't show fresh start dialog if user has previously dismissed it.
* Data Sync: Load fallback value while merging an entry

### Other information:


## Jetpack product discussion
pc9hqz-28r-p2#comment-1720

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
- Upgrade to boost premium
- Dismiss the prompt that says "Boost will now record performance"
- Refresh the page to make sure it doesn't appear again
- Ones there is initial data in the graph, make sure that the first datapoint has the graph lines continued from before the beginning of the graph.
